webui.py

```diff
@@ -17,7 +17,10 @@
     cmd_flags_path = os.path.join(script_dir, "CMD_FLAGS.txt")
     if os.path.exists(cmd_flags_path):
         with open(cmd_flags_path, 'r') as f:
-            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip())
+            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip() and not line.strip().startswith('#'))
+    if CMD_FLAGS:
```

Comment:
CMD_FLAGS may not be defined in this line. I think it's fine to remove this print statement as a whole

Reply:
Good catch - thank you.

I've fixed the error by moving the output to live within the `os.path.exists` block, ensuring CMD_FLAGS is set.

I find that having the flags displayed upon startup serves as a bit of a reassuring sanity check.  I'm happy to remove it, though, if you find it to be more noise than signal ;)

Reply:
FYI - I've tested the following cases with this PR locally:

CMD_FLAGS.txt
  1.  does not exist
  2.  is empty
  3.  has flags but no commented lines
  4.  has flags and commented lines